.github/workflows/task_e2e_tests.yml

```diff
@@ -75,13 +75,15 @@ jobs:
 
       - name: Restore dependencies
         working-directory: ./frontend
-        run: pnpm install --frozen-lockfile
+        run: |
+          pnpm install --frozen-lockfile
+          pnpm cypress install
```

Comment:
why is this extra one needed?

Reply:
They changed it in pnpm@10 

https://github.com/pnpm/pnpm/releases/tag/v10.0.0